openshift/secret.yaml.template

```diff
@@ -0,0 +1,15 @@
+# Template for secrets.yaml, for future references.
```

Comment:
I would suggest not to commit this file, since on statging/production these would be read from the common configmap of Bayesian, and devcluster setup if done via the deployer script does apply these secrets for you.